library/core/src/num/mod.rs

```diff
@@ -87,27 +87,103 @@ depending on the target pointer size.
     };
 }
 
+macro_rules! widening_impl {
+    ($SelfT:ty, $WideT:ty, $BITS:literal) => {
+        /// Calculates the full multiplication of `self` and `rhs`.
+        ///
+        /// This returns the low-order (insignificant) bits and the high-order (significant) bits
```

Comment:
Is "insignificant" the right word?

Reply:
I'm not sure, honestly-- open to suggestions. My main goal was to get an implementation out there and to work out the specifics on stuff like this after merging, though, since it has to exist in order for people to use it.

Reply:
Isn't the terminology "least significant" and "most significant" bits? [Wikipedia](https://en.wikipedia.org/wiki/Bit_numbering#Least_significant_bit).

Reply:
That is correct -- I don't know why I thought insignificant was the right term.